app/models/action.rb

```diff
@@ -2,11 +2,5 @@ class Action < ActiveRecord::Base
   belongs_to :page
   belongs_to :action_user
 
-
-  class << self
-    def create_action(params)
-      ManageAction.new(params).create
-    end
-  end
```

Comment:
This is one of my hollow abstractions. Just call `ManageAction` directly if you want it!